dashboard/app/models/pd/workshop_material_order.rb

```diff
@@ -50,7 +50,7 @@ class WorkshopMaterialOrder < ActiveRecord::Base
     has_one :workshop, class_name: 'Pd::Workshop', through: :enrollment, foreign_key: :pd_workshop_id
 
     validates :enrollment, presence: true, uniqueness: true
-    validates :user, presence: true, uniqueness: true
+    validates :user, presence: true, uniqueness: true, if: -> {user_id_changed?}
```

Comment:
Unfortunately, I think `user_id_changed?` is false for a newly created `Pd::WorkshopMaterialOrder` without a user, making the validation mostly useless? Note that I'm lacking context here.